ci/algolia-schema.json

```diff
@@ -20,7 +20,7 @@
   "unretrievableAttributes": null,
   "optionalWords": null,
   "queryLanguages": ["en"],
-  "replicas": [],
+  "replicas": ["ALGOLIA_RESEARCH_OUTPUT_INDEX_REPLICA"],
```

Comment:
if we have this setting here, do we still need to manually create and upload data to the replica? what is the purpose of this then?

Reply:
The purpose is to add the name of the replica to the primary index. 
`ALGOLIA_RESEARCH_OUTPUT_INDEX_REPLICA` is a placeholder, that will be replaced with the value of the var `$ALGOLIA_RESEARCH_OUTPUT_INDEX_REPLICA`
https://github.com/yldio/asap-hub/blob/70b4f295a89ecacf55137e26baa3007b3862afab/package.json#L19
https://github.com/yldio/asap-hub/blob/70b4f295a89ecacf55137e26baa3007b3862afab/.github/workflows/env-sync.yml#L23
https://github.com/yldio/asap-hub/blob/70b4f295a89ecacf55137e26baa3007b3862afab/.github/workflows/master-sync.yml#L25

Reply:
Shouldn't we use a better name. Its clear that its a replica what isn't clear is why the replica is different to the normal result set

Reply:
What name would you suggest?

Reply:
My first thought was:
ALGOLIA_RESEARCH_OUTPUT_INDEX_REPLICA_<fieldname>_<asc/desc>

However Algolia is redundant, Research output no longer reflects the contents of the index, Should we be calling a replica an index? 

What do we think about 
ASAP_INDEX_REPLICA_<fieldname>_<asc/desc>?

Reply:
I understand the placeholder, what i dont understand is this - we have a file `algolia-schema.json`, yeah? what is it being used for? before your changes?

Reply:
I don't think it would work. This naming convention you are proposing works if there's only one attribute. 
i.e. `ASAP_INDEX_REPLICA_addedData_desc`

But, the replica can be ranked by one or more fields. Then we can end up with names like `ASAP_INDEX_REPLICA_addedData_desc_title_asc`  or `ASAP_INDEX_REPLICA_addedDataDesc_titleAsc` 

Perhaps something like `ASAP_INDEX_REPLICA_section_REPLICA` works better. 
i.e. `ASAP_INDEX_REPLICA_ROSLISTING_REPLICA`

What do you think @kmaid ?

Reply:
> I understand the placeholder, what i dont understand is this - we have a file `algolia-schema.json`, yeah? what is it being used for? before your changes?

@peterstarling, sorry I misunderstood your comment.  
`algolia-schema.json` defines the config for the primary index. Before this PR, the config file looks like this
https://github.com/yldio/asap-hub/blob/master/ci/algolia-schema.json#L23

This PR adds a replica to the primary index
https://github.com/yldio/asap-hub/blob/70b4f295a89ecacf55137e26baa3007b3862afab/ci/algolia-schema.json#L23

And it also defines a different config for the replica. We cannot use the same config because the ranking is different
https://github.com/yldio/asap-hub/blob/70b4f295a89ecacf55137e26baa3007b3862afab/ci/algolia-replica-schema.json#L2

Reply:
ok my questions is this - do we need this here? and if so, with just this config doesnt it automatically create the replica? this is what i am getting at - i would imagine that when you sync it would create the replica too but i might be wrong

Reply:
That's a good point! I'll chat with you offline